system_tests/run_system_test.py

```diff
@@ -34,6 +35,7 @@
     'bigquery': ['project', 'credentials'],
     'bigtable': ['project', 'credentials'],
     'bigtable-happybase': ['project', 'credentials'],
+    'logging': ['project', 'credentials'],
```

Comment:
This will cause a merge conflict with the `datastore-v1beta3` branch. Remind me to merge after this goes in `master`?